x-pack/plugins/endpoint/public/applications/endpoint/view/hosts/index.tsx

```diff
@@ -137,56 +139,73 @@ export const ManagementList = () => {
       },
       {
         field: '',
-        name: i18n.translate('xpack.endpoint.management.list.lastActive', {
+        name: i18n.translate('xpack.endpoint.host.list.lastActive', {
           defaultMessage: 'Last Active',
         }),
+        dataType: 'date',
         render: () => {
           return 'xxxx';
         },
       },
     ];
   }, [queryParams, history]);
 
+  const HostPage = styled(EuiPage)`
+    padding: 0;
+  `;
+
+  const HostHeader = styled(EuiPageHeader)`
+    background-color: #f5f7fa;
```

Comment:
Do not use hardcoded values for colors/padding etc. unless they are not provided by Eui.
Styled components provide a mechanism for access to the Eui Theme which you should use here. To do that, you define a function that accepts `props` for the CSS property value and then use that to return back the definition of that css property.

Example of using a borders and colors from the Eui Theme:

```javascript
const Container = styled.div`
  border-bottom: ${props => props.theme.eui.euiBorderThin};
  background-color: ${props => props.theme.eui.euiPageBackgroundColor};
`;

const Wrapper = styled.div`
  margin-right: auto;
  padding-top: ${props => props.theme.eui.paddingSizes.xl};
  padding-left: ${props => props.theme.eui.paddingSizes.m};
  padding-right: ${props => props.theme.eui.paddingSizes.m};
`;
```